frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/executions.cy.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable camelcase */
+import {
+  buildMockPipelineV2,
+  buildMockPipelines,
+  mockDashboardConfig,
+  mockDataSciencePipelineApplicationK8sResource,
+  mockK8sResourceList,
+  mockProjectK8sResource,
+  mockRouteK8sResource,
+} from '~/__mocks__';
+import {
+  DataSciencePipelineApplicationModel,
+  RouteModel,
+  ProjectModel,
+} from '~/__tests__/cypress/cypress/utils/models';
+import {
+  executionPage,
+  executionFilter,
+} from '~/__tests__/cypress/cypress/pages/pipelines/executions';
+import { mockGetExecutions, mockGetNextPageExecutions } from '~/__mocks__/mlmd/mockGetExecutions';
+import { initMlmdIntercepts } from './mlmdUtils';
+
+const projectName = 'test-project-name';
+const initialMockPipeline = buildMockPipelineV2({ display_name: 'Test pipeline' });
+
+describe('ExecutionsError', () => {
+  it('Fails to load executions list', () => {
+    initIntercepts(false);
+    executionPage.visit(projectName);
+    cy.contains('There was an issue loading executions');
+  });
+});
+
+describe('No Executions', () => {
+  it('Has no executions', () => {
+    initIntercepts(true, true);
+    executionPage.visit(projectName);
+    cy.contains('No executions');
+  });
+});
+
+describe('Executions', () => {
+  beforeEach(() => {
+    initIntercepts(true);
+    executionPage.visit(projectName);
+  });
+
+  it('Makes filter request', () => {
+    shouldFilterItems(FilterArgs.Execution, 'h');
+    shouldFilterItems(FilterArgs.ID, '2');
+    shouldFilterItems(FilterArgs.Status);
+    shouldFilterItems(FilterArgs.Type);
+  });
+
+  it('Makes requests to include more entries', () => {
+    executionFilter.findEntriesPerPage().click();
+    setUpIntercept();
+    cy.findByText('20 per page').click();
+    cy.wait('@request');
+    executionFilter.findEntriesPerPage().click();
+    setUpIntercept();
+    cy.findByText('30 per page').click();
+    cy.wait('@request');
+  });
+
+  it('Visits execution details page', () => {
+    cy.get('a[href="/executions/test-project-name/288"]').click();
+    cy.contains('288');
+    cy.contains('system.ContainerExecution');
+  });
+
+  it('Navigates to next page and previous page', () => {
+    executionFilter.findNextPage().as('nextpage');
+    setUpInterceptForNextPage();
+    cy.get('@nextpage').click();
```

Comment:
Can we not directly do this ??
```suggestion
  executionFilter.findNextPage().click();
```

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/2968#discussion_r1663080317